core/v1.0/core-v1.0.md

```diff
@@ -0,0 +1,609 @@
+```
+
+If no schema has been linked to that name, it is interpreted as a local name:
+
+```graphql example -- a strange local name
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")  
+# 1. Schema("link") -> "https://specs.apollo.dev/link/v1.0" (explicit)
+# 2. Directive("link") -> "https://specs.apollo.dev/link/v1.0#@link" (implicit)
+
+#    👇🏽 🌍 #myOwn__Purpose
+enum myOwn__Purpose { SECURITY EXECUTION }
+```
+
+Note: GraphQL name conventions strongly suggest against such naming. But amongst the core schema design principles is *universality*—the ability to represent and link *any arbitrary* set of GraphQL schemas, no matter how weird the names in them are.
+
+## Bootstrapping
+
+Documents can {@link} link itself. Indeed, if they MUST do so if they use {@link} at all and are intended to be [fully valid](#sec-Fully-Valid-Core-Schemas):
+
+```graphql example -- bootstrapping {@link}
+extend schema
+  @link(url: "https://specs.apollo.dev/link/v1.0")
+```
+
+The {@link} directive for {@link} itself—the "bootstrap"—MUST be the first {@link} in the document. Other directives may precede it, including directives which have been {@link}ed. 
+
+There is otherwise nothing special or restricted about bootstraps. Documents MAY use them to rename {@link}—either with [`as:`](#@link.as) or [`import:`](#@link.import) or both:
+
+```graphql example -- bootstrapping {@link} with a different name
+extend schema
+  @core(url: "https://specs.apollo.dev/link/v1.0", as: "core")
+```
+
+```graphql example -- importing {@link} with a different name
+extend schema
+  @core(url: "https://specs.apollo.dev/link/v1.0", import: [{ name: "@link", as: "@core" }])
+```
+
+```graphql example -- importing the {@link} directive directly
+extend schema
+  @foo(url: "https://specs.apollo.dev/link/v1.0", import: [
+    {name: "@link", as: "@foo"},
+  ])
+```
+
+# Fully Valid Core Schemas
+
+For a document to be a fully valid core schema:
+
+1. It MUST be a valid GraphQL schema. Amongst other things, this means that it MUST contain definitions for all types and directives it references, including those from foreign schemas
+2. If it includes any {@link}s, it MUST include a [bootstrap](#sec-Bootstrapping) which MUST precede all other {@link}s in the document
+
+Good news: this means that every valid GraphQL schema which *does not* use {@link} is automatically a fully valid core schema. Otherwise-valid schemas which use {@link} simply have to include a [bootstrap](#sec-Bootstrapping) to become fully valid core schemas.
+
+Note: The bootstrap link is required in order to properly identify the [version](#sec-Versioning) of the link spec in use.
+
+## Partial Schemas
+
+Partial schemas are schemas which use {@link} to reference definitions which they do not necessarily include. Partial schemas are invalid GraphQL, but can be turned into valid GraphQL by a [compiler](#sec-Appendix-Compilation) which inserts the appropriate definitions.
```

Comment:
This got me confused at first because I was missing the distinction between including/inserting something and using `@link` (it feel weird to have both). I'd suggest something like:

```
A partial schema is a document before it is processed by a [compiler](#sec-Appendix-Compilation). 
Partial schemas may use {@link} and are invalid GraphQL but can be turned into valid GraphQL 
by a [compiler](#sec-Appendix-Compilation).
```